<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRendererRasterPropsWidgetBase</class>
 <widget class="QWidget" name="QgsRendererRasterPropsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cboRenderers"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="pageNoWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>This renderer doesn't implement a graphical interface.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="mLayerRendering">
        <property name="title">
         <string>Layer rendering</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="mBlendTypeLabel">
           <property name="text">
            <string>Blending mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="3">
          <widget class="QgsBlendModeComboBox" name="mBlendModeComboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Brightness</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QSlider" name="mSliderBrightness">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>-255</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
             <property name="tickInterval">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mBrightnessSpinBox">
             <property name="minimum">
              <number>-255</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="labelSaturation">
           <property name="text">
            <string>Saturation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QSlider" name="sliderSaturation">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxSaturation">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="decimals" stdset="0">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Contrast</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QSlider" name="mSliderContrast">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mContrastSpinBox">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="labelGrayscale">
           <property name="text">
            <string>Grayscale</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="3">
          <widget class="QComboBox" name="comboGrayscale">
           <item>
            <property name="text">
             <string>Off</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>By lightness</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>By luminosity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>By average</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Hue</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="3">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QCheckBox" name="mColorizeCheck">
               <property name="text">
                <string>Colorize</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QgsColorButtonV2" name="btnColorizeColor">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelColorizeStrength">
             <property name="text">
              <string>Strength</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QSlider" name="sliderColorizeStrength">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinColorizeStrength">
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="6" column="2" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QToolButton" name="mResetColorRenderingBtn">
             <property name="toolTip">
              <string>Reset all color rendering options to default</string>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/mActionUndo.svg</normaloff>:/images/themes/default/mActionUndo.svg</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mResamplingBox">
        <property name="title">
         <string>Resampling</string>
        </property>
        <layout class="QGridLayout" name="gridLayout2">
         <item row="0" column="0">
          <widget class="QLabel" name="mZoomedInResamplingLabel">
           <property name="text">
            <string>Zoomed in</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="mZoomedOutResamplingLabel">
           <property name="text">
            <string>Zoomed out</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="mMaximumOversamplingLabel">
           <property name="text">
            <string>Oversampling</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="mZoomedInResamplingComboBox"/>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="mZoomedOutResamplingComboBox"/>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="mMaximumOversamplingSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
           <item row="7">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsBlendModeComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsblendmodecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
